src/translate/pddl_parser/warning.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+
+PRINTED_WARNINGS = set()
```

Comment:
I wouldn't make this all-caps any more. I think this is now a more "normal" variable than it used to be.